src/index.js

```diff
@@ -254,13 +271,14 @@ class Mongo {
     await this.sessionCollection.updateMany({ userId }, {
       $set: {
         valid: false,
-        [this.options.timestamps.updatedAt]: Date.now(),
+        [this.options.timestamps.updatedAt]: this.options.dateProvider(),
       },
     });
   }
 
   findSessionById(sessionId: string): Promise<?SessionType> {
-    return this.sessionCollection.findOne({ _id: toMongoID(sessionId) });
+    const id = this.options.convertSessionIdToMongoObjectId ? toMongoID(sessionId) : sessionId;
```

Comment:
instead of naming with `id` please name the const as `_id` and then you can change the selector to `{ _id }`